### ISSUE TITLE
No output for inception_resnet_v2

### ISSUE THREAD

**Issue by collinarnett** [you359/Keras-FasterRCNN#27]
Inception resnet v2 does not produce a bbox even when predicting on training data.

**Comment by abtExp** [comment]
it's the same for xception.

**Comment by collinarnett** [comment]
I think this is because this repository does not implement validation loss, therefore the most likely reason we are not seeing a output is because the model is over-fitting.

**Comment by abtExp** [comment]
@collinarnett  The model doesn't even use the validation data in this implementation, so might be a possibility, have you tried running on one sample from the training set itself?

**Comment by collinarnett** [comment]
@abtExp I had the issue where I wasn't loading the base weights properly. When I loaded the base weights I ended up getting the correct output. I recommend also removing the try catch statement from that part of the code since you want it to fail if it can't load the base weights.

**Comment by collinarnett** [comment]
@abtExp Did that solve your issue as well?

**Comment by abtExp** [comment]
@collinarnett i'ven't tried yet. but i also see the loading of weights failing everytime, so i guess that might be the issue. I'll fix the weights loading and try again.

**Comment by collinarnett** [comment]
Ok let me know and I can close this issue

**Comment by collinarnett** [comment]
@abtExp  Just bumping this issue to the top of your inbox

**Comment by abtExp** [comment]
@collinarnett hi, sorry I haven't been able to get back on the project that I was using this in, you go ahead and close this issue if it solves the problem for you, I'll try out the fix that worked for you and if I face a problem, I'll reopen the issue. :+1:
